packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -448,6 +448,14 @@ rocksdb::Status ArbCore::reorgToMessageCountOrBefore(
         std::get<std::unique_ptr<MachineThread>>(std::move(setup));
     auto& output = new_machine->machine_state.output;
 
+    if (use_latest) {
+        std::cerr << "Loaded valid checkpoint, gas: " << output.arb_gas_used
+                  << ", messages: " << output.fully_processed_inbox.count
+                  << ", steps: " << output.total_steps
+                  << ", logs: " << output.log_count << ", sends"
+                  << output.send_count << "\n";
```

Comment:
Somewhat unrelated, but we might want to add this info to the "Last checkpoint gas used" log statement.

Reply:
Yes.  I already added additional info in more-optimizations branch, will add to that branch to reduce conflicts.